content/en/docs/setup/independent/high-availability.md

```diff
@@ -201,63 +215,40 @@ SSH is required if you want to control all nodes from a single machine.
     kubectl get pod -n kube-system -w
     ```
 
-    - It's recommended that you join new control plane nodes only after the first node has finished initializing.
+    - It's recommended that you join new control plane nodes in a sequence, only after the first node has finished initializing.
```

Comment:
Please move this point before/after the kubeadm join --experimental-control-plane command (L256 before your changes)